### ISSUE TITLE
[java] CommentDefaultAccessModifier is triggered in JUnit5 test class

### ISSUE THREAD

**Issue by koalalam** [pmd/pmd#3859]
**Affects PMD Version:** 

6.43.0

**Rule:**

[JUnit5TestShouldBePackagePrivate](https://pmd.github.io/latest/pmd_rules_java_bestpractices.html#junit5testshouldbepackageprivate)
[CommentDefaultAccessModifier](https://pmd.github.io/latest/pmd_rules_java_codestyle.html#commentdefaultaccessmodifier)

**Description:**

These 2 rules conflict with each other in JUnit5 test cases

**Code Sample demonstrating the issue:**

```java
import org.junit.jupiter.api.Test;

class SomeTest {

  @Test
  void test() {
    // run some test
  }
}
```

**Expected outcome:**

CommentDefaultAccessModifier is not triggered in JUnit5 test class

PMD reports a violation at line ..., but that's wrong. That's a false positive.

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*

Gradle

**Comment by koalalam** [comment]
Any update on this?

This another rule [DefaultPackage](https://pmd.github.io/pmd-6.45.0/pmd_rules_java_codestyle.html#defaultpackage) which is depreciated seems to handle JUnit5 correclty.

**Comment by lfalcantar** [comment]
I have a solution similar to the DefaultPackage, where the code will check if the method is annotated with a junit5 and exclude the error from the violations. let me know what you think @adangel
